portfolio.css

```diff
@@ -0,0 +1,279 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
+/* For Mobile */
+
+@media (max-width: 768px) {
+  .pop-close {
+    display: block;
+    position: absolute;
+    top: 20px;
+    right: 20px;
+    color: var(--color-primary);
+  }
+
+  .tonic-1 {
+    height: 36px;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 700;
+    font-size: 40px;
+    color: #091e42;
+  }
+
+  .card-portfolio-1 {
+    background-color: #fff;
+    border-radius: 8px;
+    box-shadow: 0 0 5px rgba(0, 0, 0, 0.3);
+    width: 343px;
+    height: 630px;
+    padding: 24px;
+    margin: 24px auto;
+    display: none;
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    overflow: hidden;
+  }
+
+  .card-portfolio-1 span {
+    float: right;
+    padding-top: 24px;
+    padding-left: 24px;
+  }
+
+  .forM-1 {
+    width: 292px;
+  }
+
+  .forD-1 {
+    display: none;
+  }
+
+  .heading-canopy-1 {
+    height: 36px;
+    font-family: 'Poppins', sans-serif !important;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .column-left-1 p {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    color: #344563;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin: 12px 0;
+  }
+
+  hr {
+    width: 303px;
+    border: 1px solid #ebecf0;
+    padding-left: 20px;
+    padding-right: 20px;
+  }
+
+  .miss {
+    display: none;
+  }
+
+  #miss {
+    display: none;
+  }
+
+  .modal-1 {
+    color: #6070ff;
+    display: flex;
+    gap: 4px;
+    padding-bottom: 20px;
+    margin-top: 20px;
+    font-size: 12px;
+  }
```

Comment:
- [ ] Kindly make sure the buttons in the popup matches that of the Figma template
Your design
![Capture2](https://user-images.githubusercontent.com/85095702/162290003-16de5013-593b-4a38-801d-63a6dbedf41c.PNG)
Figma Design
![figma](https://user-images.githubusercontent.com/85095702/162290060-fa9f042a-d8f8-46e4-912b-371911660f8e.PNG)